docs/how-ubuntu-is-made/concepts/glossary.md

```diff
@@ -277,13 +277,13 @@ Circle of Friends
     [core values of Ubuntu](https://design.ubuntu.com/brand):
     *Freedom*, *Reliable*, *Precise* and *Collaborative*.
 
-    ```{image} ../images/glossary-CoF-Square.png
+    ```{image} ../../images/glossary-CoF-Square.png
```

Comment:
Also commented on this at https://github.com/ubuntu/ubuntu-project-docs/pull/65#pullrequestreview-2952985854
```suggestion
    ```{image} /images/glossary-CoF-Square.png
```